docs/examples/adddocsref.pl

```diff
@@ -25,14 +25,15 @@
 
 # pass files as argument(s)
 
-my $docroot="https://curl.se/libcurl/c";
+my $docroot = "https://curl.se/libcurl/c";
 
 for $f (@ARGV) {
     open(NEW, ">$f.new");
     open(F, "<$f");
     while(<F>) {
         my $l = $_;
         if($l =~ /\/* $docroot/) {
+
             # just ignore preciously added refs
```

Comment:
Adding an empty line before comments seems unnecessary. Something better done manually.

Deleting dupe empty lines is nice though.